bundler/spec/commands/help_spec.rb

```diff
@@ -22,9 +22,9 @@
     expect(out).to match(/bundle-install/)
   end
 
-  it "still outputs the old help for commands that do not have man pages yet" do
-    bundle "help version"
-    expect(out).to include("Prints the bundler's version information")
+  it "still outputs the old help for commands that do not have man pages yet", :bundler => "< 3" do
+    bundle "help console"
+    expect(out).to include("Opens an IRB session with the bundle pre-loaded")
```

Comment:
Note: `bundle help console` is the very last command which depends on Thor-generated man pages and this command is to be removed in Bundler 3 so we can test this case only in Bundler 2.x.

Reply:
I think `bundle help fund` also does not have a specific man page, so we could use that instead?